solr/contrib/prometheus-exporter/src/test/org/apache/solr/prometheus/utils/Helpers.java

```diff
@@ -44,4 +45,15 @@ public static void indexAllDocs(SolrClient client) throws IOException, SolrServe
     }
     client.commit(PrometheusExporterTestBase.COLLECTION);
   }
+
+  // Parses a prometheus line into key and value, e.g.
+  //   solr_exporter_duration_seconds_bucket{le="1.0",} 1.0
+  //   first="solr_exporter_duration_seconds_bucket{le="1.0",}," second=1.0
+  public static Pair<String, Double> parseMetricsLine(String line) {
+    int spaceIdx = line.lastIndexOf(" ");
+    if (spaceIdx == -1) {
+      throw new RuntimeException("Failed parsing metrics line, must contain a space");
```

Comment:
nit: I think this could be an illegal argument exception to be more specific and also include the line that we failed to parse.

Reply:
Fixed